src/core/search/indices.cc

```diff
@@ -43,12 +47,14 @@ absl::flat_hash_set<std::string> TokenizeWords(std::string_view text) {
 }
 
 // Split taglist, remove duplicates and convert all to lowercase
-absl::flat_hash_set<string> NormalizeTags(string_view taglist) {
+absl::flat_hash_set<string> NormalizeTags(string_view taglist, bool case_sensitive,
+                                          char separator) {
   string tmp;
   absl::flat_hash_set<string> tags;
-  for (string_view tag : absl::StrSplit(taglist, ',')) {
+  for (string_view tag : absl::StrSplit(taglist, separator, absl::SkipEmpty())) {
     tmp = absl::StripAsciiWhitespace(tag);
-    absl::AsciiStrToLower(&tmp);
+    if (!case_sensitive)
+      absl::AsciiStrToLower(&tmp);
     tags.insert(std::move(tmp));
```

Comment:
> The only two transformations performed are lower-casing (for latin languages only as of now)

Need to check how tags work with unicode to be on the safe side. So far I assumed they're only used with ascii 🤷🏻‍♂️ Can you split by a character (and not by a byte), etc...